### PR TITLE
lspci: add page

### PR DESCRIPTION
**Pull request by RTFMExe**
----

- [X] The page (if new), does not already exist in the repo.

- [X] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [X] The page has 8 or fewer examples.

- [X] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [X] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/linux/lspci.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+`lspci -v`
+
+- Display drivers and modules handling each device:
+
+`lspci -k`
+
+- Show a specific device:
+
+`lspci -s {{bus:device.func}}`
```
Comment (sbrl) [inline pages/linux/lspci.md]: Where can I get this information? Perhaps a concrete example here would work better?

File: pages/linux/lspci.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Dump info in readable form:
```
Comment (sbrl) [inline pages/linux/lspci.md]: Perhaps `in a readable` would read better?

Comment (RTFMExe) [reply pages/linux/lspci.md]: I agree.

File: pages/linux/lspci.md (hunk 1 above)
Comment (RTFMExe) [reply pages/linux/lspci.md]: Hi, If you run lspci you get lines like this:
**01:0a.0** Ethernet controller: Realtek Semiconductor Co., Ltd. RTL-8100/8101L/8139 PCI Fast Ethernet Adapter (rev 10)

The highlighted portion shows the bus, device and function values. Yeah maybe concrete example would work better.

Comment (sbrl) [reply pages/linux/lspci.md]: Ahhh ok.

**Comment by sbrl** [comment]
Thanks, @RTFMExe 😺

**Pull request merged**